app/controllers/api/api_controller.rb

```diff
@@ -40,17 +42,28 @@ def render_error(status, message)
     # part of Rails' instrumentation code, and is run after each request.
     def append_info_to_payload(payload)
       super
-
-      payload[:custom_log_items] = {
-        request_id: RequestStore.store[:request_id]
-      }
+      payload[:custom_log_items] = Instrumentation.custom_log_items
     end
 
     def store_request_id
       RequestStore.store[:request_id] = request.uuid
+      Instrumentation.append_to_log(request_id: RequestStore.store[:request_id])
       Raven.extra_context(request_id: RequestStore.store[:request_id])
     end
 
+    def log_api_counters
+      Instrumentation.append_to_log(api_request_count: api_request_count)
+      Instrumentation.append_to_log(api_error_count:   api_error_count)
+    end
+
+    def api_request_count
+      RequestStore.store[:api_request_count]
+    end
+
+    def api_error_count
+      RequestStore.store[:api_error_count]
+    end
+
```

Comment:
I find this implementation a bit low level. Rather than handing each incrementer separately, what about adding a method which increments a counter, and handles appending to the log internally?

    Instrumentation.incr(:api_request_count)